nqeit-iceberg-dremio-extension/src/main/java/org/projectnessie/integtests/dremio/Dremio.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2023 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.integtests.dremio;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.PARAMETER)
+@Retention(RetentionPolicy.RUNTIME)
+@Inherited
+public @interface Dremio {
+
+  String systemPropertyPrefix() default "dremio.";
```

Comment:
Why do we have "override" prefixes in Spark and Flink, but a prefix for the full set of property in Dremio? I see that Dremio setup code is different, but why not have consistency at the test code level? :thinking:

Reply:
fair enough